source/includes/connection-snippets/scram-snippets/php-connection.php

```diff
@@ -0,0 +1,10 @@
+<!-- begin scram connection -->
+<?php
+
+$client = new MongoDB\Client(
```

Comment:
Nitpick: I think usually you indent within a php block

Reply:
Fixed - indentation. Thanks!